src/instructlab/config/init.py

```diff
@@ -320,14 +327,17 @@ def match_profile_based_on_gpu_count(
 
 def check_if_configs_exist(fresh_install) -> bool:
     if exists(DEFAULTS.CONFIG_FILE):
-        overwrite = click.confirm(
-            f"Found {DEFAULTS.CONFIG_FILE}, do you still want to continue?"
-        )
+        click.echo(f"Existing config file was found in:\n    {DEFAULTS.CONFIG_FILE}")
+        overwrite = click.confirm("Do you still want to continue?")
         if not overwrite:
             raise click.exceptions.Exit(0)
     if exists(DEFAULTS.TRAIN_PROFILE_DIR) and not fresh_install:
+        click.echo()
+        click.echo(
+            f"Existing training profiles were found in:\n    {DEFAULTS.TRAIN_PROFILE_DIR}"
+        )
         return click.confirm(
-            f"Existing training profiles were found in {DEFAULTS.TRAIN_PROFILE_DIR}\nDo you want to restore these profiles to the default values?"
+            "Do you want to restore these profiles to the default values?"
```

Comment:
Same comment here that I am not sure whether the newline could cause some people to not make the connection.